workspace.go

```diff
@@ -65,6 +67,15 @@ type Workspaces interface {
 
 	// UnassignSSHKey from a workspace.
 	UnassignSSHKey(ctx context.Context, workspaceID string) (*Workspace, error)
+
+	// Add remote state consumers to a workspace.
+	AddRemoteStateConsumers(ctx context.Context, workspaceID string, options WorkspaceAddRemoteStateConsumersOptions) error
+
+	// Remove remote state consumers from a workspace.
+	RemoveRemoteStateConsumers(ctx context.Context, workspaceID string, options WorkspaceRemoveRemoteStateConsumersOptions) error
+
+	// ReadRemoteStateConsumers reads the remote state consumers for a workspace.
+	ReadRemoteStateConsumers(ctx context.Context, workspaceID string) (*WorkspaceList, error)
```

Comment:
Looking across the other APIs, I think "fetching the thing" is usually not prefixed with `Read`/`Fetch`/etc; seems like this should be just `RemoteStateConsumers` for consistency.

Reply:
Updated to be `RemoteStateConsumers`